packages/cli/templates/default-app/.eslintrc.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  extends: [require.resolve('@backstage/cli/config/eslint')],
+  root: true,
```

Comment:
Wait what no? :grin:

If people want to use their own config we're not stopping them, but I think we should default to the one we use so that we keep code familiar across repos

Reply:
How to separate between backend and frontend config otherwise? got a react-hook-rule error in our backend HMR support :-D

Reply:
Oh this is actually the root one in the new app 😅, yeah should've always been root so 👍